source/fundamentals/crud/read-operations/text.txt

```diff
@@ -36,12 +45,11 @@ snippet:
    :start-after: begin insert docs
    :end-before: end insert docs
 
-.. include:: /includes/fundamentals/automatic-db-coll-creation.rst
-
-Each document contains the name and release year of the Marvel movie
-that corresponds to the ``title`` and ``year`` fields.
+Each document contains the name and description of an dish on a
+restaurant's menu. These items correspond to the ``name`` and
```

Comment:
Issue: Is there a more technical name than "items"? I'm not 100% on what this is referring to.

Reply:
Changed to be more concise because it feels self explanatory to me, but let me know if you think it should be two sentences. 

`Each document contains the ``name`` and ``description`` of a dish on a restaurant's menu.`